march_rqt_input_device/src/march_rqt_input_device/input_device.py

```diff
@@ -184,6 +184,11 @@ def __init__(self, context):
                                                       callback=lambda: self.publish_gait(
                                                           "gait_rough_terrain_middle_steps"))
 
+        gait_ramp_door_slope_up = MarchButton(name="gait_ramp_door_slope_up",
+                                                      text="MV Ramp and Door slope up",
+                                                      callback=lambda: self.publish_gait(
+                                                          "gait_ramp_door_slope_up"))
```

Comment:
You have to indent this section correctly:
```python
gait_ramp_door_slope_up = MarchButton(name="gait_ramp_door_slope_up",
                                      text="MV Ramp and Door slope up",
                                      callback=lambda: self.publish_gait("gait_ramp_door_slope_up"))
```

Reply:
indeed, clion has an automated function to do this: <ctrl+alt+L>
https://www.jetbrains.com/help/clion/reformat-and-rearrange-code.html